opts/network.go

```diff
@@ -0,0 +1,100 @@
+package opts
```

Comment:
Yes. I agree. There are multiple other CLIs with their CSV options all part of the `opts` package and are currently used only in docker/cli. I guess moving `opts` out to `docker/cli` is its own independent work.